Source/Atomic/Core/Object.cpp

```diff
@@ -302,9 +305,35 @@ void Object::SendEvent(StringHash eventType)
 
     SendEvent(eventType, noEventData);
 }
-
+// ATOMIC BEGIN
 void Object::SendEvent(StringHash eventType, VariantMap& eventData)
 {
+#if ATOMIC_PROFILING
+    bool eventProfilingEnabled = false;
+    if (Profiler* profiler = GetSubsystem<Profiler>())

```

Comment:
Thanks to magic template specializations it gets profiler object from `context_` anyway, just the same as if new getter was used :p Yes, we get to use new getters for free without changing any existing code. Much 🐫